src/openrct2-ui/windows/Ride.cpp

```diff
@@ -4353,7 +4353,7 @@ static void window_ride_maintenance_paint(rct_window* w, rct_drawpixelinfo* dpi)
             }
             else
             {
-                auto peep = (&(get_sprite(ride->mechanic)->peep))->AsStaff();
+                auto peep = get_sprite(ride->mechanic)->As<Peep>()->AsStaff();
```

Comment:
It would be great if one could just do 
```suggestion
                auto peep = get_sprite(ride->mechanic)->As<Staff>();
```